> Interleaved image test should pass now.

I ran the tests in my local env and get the following errors:
```
FAILED test_mllama.py::test_models_interleaved_images[_Backend.XFORMERS-5-128-bfloat16-meta-llama/Llama-3.2-11B-Vision-Instruct] - AttributeError: 'list' object has no attribute 'shape'
FAILED test_mllama.py::test_models_interleaved_images[_Backend.FLASH_ATTN-5-128-bfloat16-meta-llama/Llama-3.2-11B-Vision-Instruct] - AttributeError: 'list' object has no attribute 'shape'
```
I'm expecting this PR to fix it https://github.com/vllm-project/vllm/pull/14883.

How can you pass the interleaved image tests?

--- a/vllm/model_executor/models/mllama.py
+++ b/vllm/model_executor/models/mllama.py
@@ -214,7 +214,7 @@ class MllamaMultiModalProcessor(EncDecMultiModalProcessor[MllamaProcessingInfo]
         if mm_data:
             # Since only the last group of consecutive images
             # are attended by the decoded tokens, we only need to
-            # get the number of tiles for those images.
+            # get the number of tokens for those images.
             token_per_chunk = self.info.get_token_per_chunk_from_config()
             num_decode_images = self._get_num_image_in_last_group(
                 mm_inputs["prompt_token_ids"])

